lib/inferno/dsl/fhir_evaluation/rules/all_resources_reachable.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../../../utils/evaluator_util'
+
+module Inferno
+  module DSL
+    module FHIREvaluation
+      module Rules
+        class AllResourcesReachable < Rule
+          attr_accessor :config
+
+          def check(context)
+            @config = context.config
+            # TODO: add some customizable configurations for this rule:
```

Comment:
I don't think all these TODOs belong here unless they are planned to be completed in the next couple tickets.

Reply:
Removed